torch/utils/inference/run.py

```diff
@@ -0,0 +1,569 @@
+r"""
+This is a script for launching PyTorch inference on Intel(R) Xeon(R) Scalable Processors with optimal configurations.
```

Comment:
Correct me if I'm wrong here, but aren't the settings that are configured by this launcher also apply to training? I'm wondering why we are limiting these settings to inference only.

Reply:
Yes, they should also work for training on CPUs. This time we focus on inference functionality. We plan to file a separate PR for training later.